recipes/ua-nodeset/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class UaNodeSetConan(ConanFile):
+    name = "UA-Nodeset"
+    license = ""
+    author = 'OPCFoundation'
+    description = "This repository contains UANodeSets and other normative files which are released with a specification"
+    url = 'https://github.com/OPCFoundation/UA-Nodeset'
+    default_user = "ar"
+    default_channel = "thirdparty"
+    generators = "cmake", "virtualenv"
+    settings = 'os'
+
+
+    def source(self):
+        git = tools.Git()
+        git.clone(
+            url="https://github.com/OPCFoundation/UA-Nodeset.git",
+            branch="v{}".format(self.version))
+
+    def build(self):
+        pass
+ 
+
+    def package(self):
+        self.copy("*", dst="ua-nodeset", src="")
```

Comment:
The root folders are [limited due to a hooks](https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#kb-h013-default-package-layout).
So copy it to a subdirectory instead.
```suggestion
        self.copy("*", dst="res")
```